### PR TITLE
KMA-6 - pipeline deployments for scale of belief lambda

### PR DESCRIPTION
**Pull request by mattdrees**
This PR creates deployment jobs for scale-of-belief-lambda and scale-of-belief-import. These are both using a shared global library, which is added in this PR, to define the build pipeline.

I've tried to make the elements of the pipeline somewhat decoupled from serverless-specific deployments, because I want to eventually switch many of our other jobs over to using Jenkins pipelines, as well.

The shared library here will be referenced by the Jenkinsfiles in each of those projects:
 * https://github.com/CruGlobal/scale-of-belief-lambda/blob/kma-6-jenkins-deployment/Jenkinsfile
 * https://github.com/CruGlobal/scale-of-belief-import/blob/kma-6-add-jenkins-deployments/Jenkinsfile

After this PR is merged, I'll update those files to reference the master branch in this repo, instead of this topic branch.

This PR depends on some jenkins global config done in https://github.com/CruGlobal/cru-ansible/pull/279.

This implements most of https://jira.cru.org/browse/KMA-6.

### CODE REVIEW

**Comment by mattdrees** [comment]
@SeriousMatt can I ask you for another code review? This is mostly groovy, which I'm guessing is new to you, but it's not hard. (Kind of a mixture of java and javascript.)

The docs for jenkins pipeline libraries, which I had to reference quite a bit, are here: https://jenkins.io/doc/book/pipeline/shared-libraries/

File: jobs/pipeline-jobs.yml [hunk 1]
```diff
@@ -1,47 +1,49 @@
 
-- job:
-    name: familylife-forms-api
+- project:
+    name: FamilyLife-FormsAPI
```
Comment (mattdrees) [inline jobs/pipeline-jobs.yml]: @vdymna-fl 
Do you mind if I change the two FL job names to match their github repo names? See the commit message here for the reason.

familylife-forms-api -> FamilyLife-FormsAPI
familylife-aop-api -> FamilyLife-ArtOfParentingAPI

Comment (vdymna-fl) [reply jobs/pipeline-jobs.yml]: @mattdrees yes, you can rename them for sure.

**Comment by SeriousMatt** [comment]
Will see about getting this rolled out to production this week.

**Comment by mattdrees** [comment]
Thanks Matt!

**Pull request merged**